liferay-tango/liferay-portal
# LPS-116288 Partially Revert info panel, since it is going added with a new frontend component

***Motivation***

Rever info panel previous work, since the content dashboard side panel is going o be added using a React Component

***Proposed Solution***

Revert all the previous commits

***How to test it***

1. Add a web content
2. Go to Content Dashboard
3. NO check is available in the row, NO info icon is shown.

/cc @carloslancha
